src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -281,6 +281,24 @@ URI getUrlAttribute(org.w3c.dom.Node attribute) throws InvalidInputException {
     }
   }
 
+  /**
+   * Checks if attribute has Boolean values
+   * This is done in cases we want to enforce valid values for the Boolean attributes.
+   *
+   * @param attributeName name of attribute that will be checked.
+   * @throws InvalidInputException
+   */
+  void assertAttributeHasBooleanValue(String attributeName) throws InvalidInputException {
```

Comment:
I think this can be made more general: _assertAttributeText(Collection<String> permittedValues)_. You're only comparing strings against a permitted list, after all.

Also, there's no unit test for this method.

Reply:
Seems good, will change accordingly